client/core/core.go

```diff
@@ -2993,6 +2997,12 @@ func (c *Core) prepareTrackedTrade(dc *dexConnection, form *TradeForm, crypter e
 		return nil, 0, fmt.Errorf("wallet %v failed to sign coins: %w", wallets.fromAsset.ID, err)
 	}
 
+	commitSig := make(chan struct{})
+	defer close(commitSig)
```

Comment:
Whether `signAndRequest` succeeds or fails, which was the trouble before with `piSyncers`, which was keyed on order ID, a value that was only *available* after a *successful* request/response, two problematic conditions.